lntest/harness.go

```diff
@@ -361,6 +361,8 @@ func (h *HarnessTest) Stop() {
 		return
 	}
 
+	h.shutdownAllNodes()
```

Comment:
Good catch, but I'm wondering why the subtest cleanup did not take care of the standby nodes?

Reply:
During a subtest the standby nodes are just _reset_, meaning their cached numbers (channels, UTXOs and so on) are reset, but the nodes aren't actually terminated. I think this has been this way for a while now, we just never noticed (or we did and didn't investigate).

Reply:
actually I just realized we should only shutdown standby nodes here - the non-standbys have already been shut down during the cleanup in `subtest`

Reply:
Yeah, but there is no `shutdownStandbyNodes()`, since `shutdownNodes` just takes a boolean `skipStandby`.
And if there are no non-standby nodes, shutting down all of them results in the same thing: just shutting down the standby ones.
Or would you like me to add a `shutdownStandbyNodes()` function?

Reply:
Okay double checked, indeed the subtest cleanup missed shutdown of standby nodes if the test succeeded.

Reply:
think I recalled why I wrongfully [removed it](https://github.com/lightningnetwork/lnd/commit/623f816f8e348f20460cf4e149647d1718e2154c) in the first place - when the test fails, we'll shut down the node twice, resulting in some error msgs being printed.

> Or would you like me to add a shutdownStandbyNodes() function?

Nah I think you are right - there's map that tracks the active node, and we remove it from the map whenever we shut it down, so no need for this func.

Reply:
Hmm, I just tried this out by causing a test to fail and I didn't see duplicate or weird error messages... Maybe that was fixed in another way in the meantime?

Reply:
yeah likely